Packs/jamf/Integrations/jamfV2/jamfV2.py

```diff
@@ -88,15 +111,18 @@ def _generate_token(self) -> str:
     def _classic_api_post(self, url_suffix, data, error_handler):
         post_headers = ((self._headers or {}) | POST_HEADERS)  # merge the token and the POST headers
         classic_url_suffix = urljoin('/JSSResource', url_suffix)  # classic API endpoints starts with JSSResource
+        # if token is not available, use basic auth directly
+        auth = (self.username, self.password) if (not self._token) and self.basic_auth_flag else None
```

Comment:
This should also be handled in a function handling the authentication method.

Reply:
Done